config/routes.rb

```diff
@@ -1,6 +1,22 @@
 Rails.application.routes.draw do
-  root to: "wishlists#index"
   resources :wishlists, only: [:index, :new, :create]
-  devise_for :users
+  devise_for :users, skip: [:sessions, :registrations]
```

Comment:
Everything works for me without this line. Test it yourself. If everything is OK, then I'd remove it.